test/inject.test.js

```diff
@@ -458,14 +458,14 @@ test('should handle errors in builder-style injection correctly', async (t) => {
 })
 
 test('Should not throw on access to routeConfig frameworkErrors handler - FST_ERR_BAD_URL', t => {
-  t.plan(6)
+  t.plan(5)
 
   const fastify = Fastify({
     frameworkErrors: function (err, req, res) {
       t.ok(typeof req.id === 'string')
       t.ok(req.raw instanceof Readable)
       t.same(req.routerPath, undefined)
-      t.same(req.routeConfig, undefined)
+      // t.same(req.routeConfig, undefined)
```

Comment:
Left over?

Reply:
Yeah it seems like the changes I made to onBadUrlContext (making it a proper context) has made some of the recent https://github.com/fastify/fastify/pull/4826 obsolete (it seems part of that PR was to fix parts of the context being undefined because it wasn't a proper context, now it is a proper context).

This test was added in that PR (to assert the context is not a full context?), but not it is a full context so that test fails and I had to remove it.

Shall I to remove it rather than comment it?

Reply:
It was mostly to test that it does not throw when the `req.routeConfig` is `undefined`, although now is not the case. It should be fine just removing it 👍

Reply:
Done